src/components/popover-container/popover-container.style.ts

```diff
@@ -32,29 +57,12 @@ const PopoverContainerContentStyle = styled.div<PopoverContainerContentStyleProp
   position: absolute;
   z-index: ${({ theme }) => theme.zIndex.popover};
 
-  ${({ animationState }) => {
-    switch (animationState) {
-      case "entering":
-        return `
-        opacity: 0;
-        transform: translateY(-8px);
-      `;
-      case "entered":
-        return `
-        opacity: 1; 
-        transform: translateY(0);
-        transition: all 0.3s cubic-bezier(0.25, 0.25, 0, 1.5);
-          `;
-      case "exiting":
-        return `
-        opacity: 0; 
-        transform: translateY(-8px);
-        transition: all 0.3s cubic-bezier(0.25, 0.25, 0, 1.5);
-          `;
-      default:
-        return "opacity: 0";
-    }
-  }}
+  @media screen and (prefers-reduced-motion: no-preference) {
```

Comment:
**question**: is this media-query in the CSS actually necessary? It seems redundant to me, since we're already setting `disableAnimation` to `true` if the user prefers reduced motion (although as in the the JS file, shouldn't this query be for `reduce` rather than `no-preference`?). If so then on line 65 we're already setting `opacity: 1`.

(It also occurred to me that it's better to use CSS than JS given the choice, so maybe we could remove the media query from the JS file and just use this instead. But I don't think it will as then if `disableAnimation` is false then line 65 will end  up turning the animation on. It might work if you wrap the `animationToRender` call in the appropriate media query, but feel free to not do that as I'm not sure there's much gain.)

Reply:
I had thought originally that if I could go with a CSS only approach but figured I'd need a prop to allow devs to control if they want the animation to happen or not too. I'll just make the change [you've suggested here](https://github.com/Sage/carbon/pull/6486#discussion_r1440246342)